test/servico/cst14Test.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package servico;
+
+import datamapper.PopulateDB;
+import datamapper.exceptions.NonexistentEntityException;
+import java.text.ParseException;
+import java.util.LinkedList;
+import java.util.List;
+import modelo.AusenciaModel;
+import modelo.ProfessorModel;
+import org.junit.After;
+import org.junit.AfterClass;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ *
+ * @author GUSTAVO
+ */
+public class cst14Test {
+
+    public cst14Test() {
+
+    }
+
+    @BeforeClass
+    public static void setUpClass() throws NonexistentEntityException, Exception {
+        PopulateDB.recreateDB("prosub", "root", "");
+        PopulateDB.populateUseCaseTest();
+    }
+
+    @AfterClass
+    public static void tearDownClass() {
+    }
+
+    @Before
+    public void setUp() throws ParseException, NonexistentEntityException {
+        //Declaração de Ausência do Professor1 na Segunda, horário 20:00 - 23:00
+        ProfessorService professorService = new ProfessorService();
+        ProfessorModel professor = professorService.obterProfessorPorNome("Professor1");
+        NotificacaoService notificaçãoService = new NotificacaoService();
+        notificaçãoService.notificarAusencia(professor.id, "06/06/2016 20:00", "06/06/2016 23:00", "", new LinkedList<String>());
+
+    }
+
+    @After
+    public void tearDown() {
+    }
+
+    @Test
+    public void testeCancelarAusenciaConfirma() {
+        LoginService loginService = new LoginService();
+        assertTrue(loginService.VerificarUsuarioESenha("Professor1", "3221"));
+
+        NotificacaoService notificaçãoService = new NotificacaoService();
+        List<AusenciaModel> ausencias = notificaçãoService.listarAusenciasPorProfessor("Professor1");

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 101).](https://www.codacy.com/app/calebepb/substituicao/file/2738517528/issues/source?bid=3308597&fileBranchId=3353506#l63)